src/backend/base/langflow/components/nvidia/nvidia_ingest.py

```diff
@@ -15,7 +22,8 @@ class NvidiaIngestComponent(BaseFileComponent):
     try:
         from nv_ingest_client.util.file_processing.extract import EXTENSION_TO_DOCUMENT_TYPE
 
-        VALID_EXTENSIONS = list(EXTENSION_TO_DOCUMENT_TYPE.keys())
+        # NOTE: HTML currently not supported, so manually removing it
+        VALID_EXTENSIONS = [ext for ext in EXTENSION_TO_DOCUMENT_TYPE.keys() if ext != "html"]
```

Comment:
@jordanrfrazier since this is failing ruff check anyway, maybe we should just hardcode this to the published list of supported file types on the nv-ingest [repo](https://github.com/NVIDIA/nv-ingest/blob/main/README.md):

- pdf
- docx
- pptx
- jpeg
- png
- svg
- tiff
- txt

This same list shows up in the docs